microsoft-edge/devtools-guide-chromium/workspaces/index.md

```diff
@@ -56,236 +52,459 @@ The tutorial steps below walk you through this environment setup.
 
-If you want to save a change to the webpage HTML, edit the HTML in the **Sources** tool with a workspace defined (in the **Filesystem** tab), rather than changing the HTML in the DOM tree in the **Elements** tool.
+If you want to save a change to the webpage HTML, edit the HTML in the **Sources** tool with a workspace defined (in the **Workspace** tab), rather than changing the HTML in the DOM tree in the **Elements** tool.
 
-1. Continuing from above, click the **Sources** (![Sources tool icon.](./index-images/sources-tool-icon.png)) tab.
+1. Continuing from above, in DevTools, click the **Sources** (![Sources tool icon.](./index-images/sources-tool-icon.png)) tab.
 
-1. In the **Navigator** pane on the left, select `index.html`.  The HTML for the page opens.
+1. In the **Navigator** pane on the left, in the **Workspace** tab (grouped with the **Page** tab), select `index.html`.
 
-1. Replace `<h1>DevTools Workspaces Demo</h1>` with `<h1>I Love Cake</h1>`, in the file listing (as opposed to the DOM tree in the **Elements** tool).
+   The **index.html** tab opens in the **Sources** tool.
 
-1. Press **Ctrl+S** (Windows, Linux) or **Command+S** (macOS) to save the change.
+1. In the the `<h1>` tag, replace "DevTools Workspaces Demo" with "I Love Cake".
+
+   An asterisk appears next to `index.html`.
 
-1. Refresh the page.  The heading in the rendered page changes to "I Love Cake", because that string was saved in `index.html` in your mapped `app` directory:
+1. Press **Ctrl+S** (Windows, Linux) or **Command+S** (macOS).
 
-   ![Change HTML from the Sources tool](./index-images/workspaces-workspaces-demo-sources-page-h1.png)
+   Your change is saved, and the asterisk disappears.
 
-1. Open the copy of `index.html` that's in your `app` directory in a text editor, such as Visual Studio Code.
+   You made the change in the file listing, rather than in the DOM tree in the **Elements** tool.
 
-   The `<h1>` element contains the new text, because you made the change using the **Sources** tool's editor to edit `index.html` and then saved the change, and that file was mapped in a workspace (the **Filesystem** tab), indicated by a green dot on the file's icon.
+1. Refresh the page.  The heading in the rendered page changes to "I Love Cake", because that string was saved in `index.html` in your mapped `/Demos/workspace/` directory:
+
+   ![Change HTML from the Sources tool](./index-images/sources-page-h1.png)
+
+1. In the **Sources** tool, right-click `index.html`, and then select **Open in containing folder**.
+
+   File Explorer or Finder opens.
+
+1. Open the copy of `index.html` that's in your `/Demos/workspace/` directory in a text editor, such as Visual Studio Code.
+
+   The `<h1>` element contains the new text, "I Love Cake", because you made the change using the **Sources** tool's editor to edit `index.html` and then saved the change, and that file was mapped in a workspace (the **Workspace** tab), indicated by a green dot on the file's icon.
+
```

Comment:
We might also want to mention the fact that many developers actually don't write HTML directly, ever. They use server-side languages (Java, Python, PHP), or site generators (Eleventy), or even client-side frameworks that use JavaScript code, and other markup languages than HTML.

So, in reality, very few devs will want (or have the opportunity) to change HTML files in the Workspace tab. They will most likely edit another souce file (maybe a React component JS source file), then rebuild, then reload (both of which may happen automatically for them).

Reply:
```
   Many developers don't write HTML directly, but instead use approaches such as:
   * A server-side language, such as Java, Python, or PHP.
   * A site generator, such as Eleventy.
   * A client-side framework that uses JavaScript code.
   * A markup language other than HTML, such as Markdown.

   This means that in practice, few developers change HTML files in the **Workspace** 
tab.  Most devs produce HTML files by using a different type of source file, such as a React 
component JS source file, and then rebuild and reload (which may happen automatically).
```